src/wx/strutils.cpp

```diff
@@ -29,11 +34,44 @@ wxArrayString str_split(const wxString& text, const wxString& sep, bool empty_to
     return tokens;
 }
 
-wxArrayString str_split_with_sep(const wxString& text, const wxString& sep)
+wxArrayString split_with_sep(const wxString& text, const wxString& sep)
 {
-    return str_split(text, sep, true);
+    return split(text, sep, true);
 }
 
-size_t vec_find(wxArrayString& opts, const wxString& val) {
-    return opts.Index(val);
+wxString trim(const wxString& in_str)
+{
+    std::wstring res{in_str};
+
+    res = std::regex_replace(res, std::wregex{L"^\\s+"}, L"");
+    res = std::regex_replace(res, std::wregex{L"\\s+$"}, L"");
```

Comment:
All of these use regex, which is pretty slow. But see my other comment, we may not need these.